pkgs/by-name/fe/fermi2/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  zlib,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "fermi2";
+  version = "0.1-unstable-2021-05-21";
+
+  src = fetchFromGitHub {
+    owner = "lh3";
+    repo = "fermi2";
+    rev = "cb1410972b2bd330883823116931ae67ead8b30f";
+    hash = "sha256-jDn1OBuGWDMEHI5A3R9meOykEGM6yjItSnUpx36DxgA=";
+  };
+
+  buildInputs = [ zlib ];
+  installPhase = ''
```

use default compiler for platform. this fixes platforms that use clang and not gcc
```suggestion
  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
  installPhase = ''
```